origin.spec

```diff
@@ -0,0 +1,407 @@
+#debuginfo not supported with Go
+%global debug_package %{nil}
+%global gopath      %{_datadir}/gocode
+%global import_path github.com/openshift/origin
+%global kube_plugin_path /usr/libexec/kubernetes/kubelet-plugins/net/exec/redhat~openshift-ovs-subnet
+%global sdn_import_path github.com/openshift/openshift-sdn
+
+# docker_version is the version of docker requires by packages
+%global docker_verison 1.6.2
+# tuned_version is the version of tuned requires by packages
+%global tuned_version  2.3
+# openvswitch_version is the version of openvswitch requires by packages
+%global openvswitch_version 2.3.1
+# %commit and %ldflags are intended to be set by tito custom builders provided
+# in the rel-eng directory. The values in this spec file will not be kept up to date.
+%{!?commit:
+%global commit 86b5e46426ba828f49195af21c56f7c6674b48f7
+}
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+# ldflags from hack/common.sh os::build:ldflags
+%{!?ldflags:
+%global ldflags -X github.com/openshift/origin/pkg/version.majorFromGit 0 -X github.com/openshift/origin/pkg/version.minorFromGit 0+ -X github.com/openshift/origin/pkg/version.versionFromGit v0.0.1 -X github.com/openshift/origin/pkg/version.commitFromGit 86b5e46 -X k8s.io/kubernetes/pkg/version.gitCommit 6241a21 -X k8s.io/kubernetes/pkg/version.gitVersion v0.11.0-330-g6241a21
+}
+
+%if "%{dist}" == ".el7ose"
+%global package_name atomic-openshift
+%global product_name Atomic OpenShift
+%else
+%global package_name origin
+%global product_name Origin
+%endif
```

Comment:
I may pull this out and we'll simply patch AOS to update the Name: because I can't figure out how to set dist properly when running tito release so it always tries to push to origin dist-git repo.